leistungsbeschreibung/README.md

```diff
@@ -1,57 +1,57 @@
+Die fiskaltrust.Middleware stellt eine länderübergreifende Schnittstelle zur Verfügung, die über verschiedene Technologien (WCF, REST, gRPC, Serial-stream oder TCP-stream protocol) für Ihr Kassensystem erreichbar ist.
 
-Die fiskaltrust-Middleware verarbeitet einige der wichtigsten Datenfelder auf dem Beleg. Die Belegnummer als eindeutige Identifikation eines von der Registrierkasse übermittelten Beleges wird von der fiskaltrust-Middleware erstellt, um sicherzustellen, dass jeder Beleg von ihr verarbeitet wird.
+![Überblick über die Funktionsweise von fiskaltrust](../product-service-description/compliance-as-a-service/media/overview-pos-ft-middleware.png)                                
 
-Die Konformität wird durch die Kombination mehrerer Methoden und Komponenten erreicht.
+Die fiskaltrust.Middleware wird zu einem wesentlichen Bestandteil Ihres Kassensystems und gewährleistet damit die Konformität als Registrierkasse. Sie erzeugt die Belegnummer als eindeutige, fortlaufende Identifikation eines von der Registrierkasse übermittelten Beleges und stellt somit nachvollziebar dar, dass jeder Beleg von ihr verarbeitet wurde.
 
-Zunächst stellt die fiskaltrust-Middleware sicher, dass alle Belege von einem Dritten neben dem Poscreator und dem Posoperator verarbeitet werden. Dies ist die organisatorische Implementierung der Sicherheit.
+Die Konformität wird durch die Kombination diverser Methoden und Komponenten erreicht.
 
-Als technische Implementierung der Sicherheit wird jeder Request und auch jede Response gehasht, wodurch die Datenintegrität der Anforderungen und Antworten sichergestellt wird. Um Unveränderlichkeit zu gewährleisten, wird ein weiterer Hash-Wert generiert, der sich auf den gesamten Anforderungs-Antwort-Zyklus bezieht, einschließlich der Identifizierung des Zyklus, des Betriebszeitpunkts, der vom Menschen lesbaren Belegnummer und der Hash-Werte von Anforderung, Antwort und vorherigem Empfang, der so genannte Belegs-Hash-Wert. Diese Verkettung des Empfangs-Hash-Werts bietet Unveränderlichkeit und bietet die Möglichkeit, alle Änderungen und auch das Löschen bei jeder vom Possystem bereitgestellten Aktion zu erkennen.
+Zunächst wird über die fiskaltrust.Middleware sichergestellt, dass alle Belege von einer dritten Partei (fiskaltrust) neben dem Kassenhersteller und dem Kassenhändler verarbeitet werden. Dies nennen wir die organisatorische Implementierung des Schutzes vor Manipulation.
 
+Zur technische Implementierung des Schutzes vor Manipulation wird jeder Request von dem Kassensystem und auch jeder Response von der fiskaltrust.Middleware gehasht, wodurch die Datenintegrität der Requests und Responses sichergestellt wird. Um nachweisen zu können, dass keine Manipulation stattgefunden hat, wird ein weiterer Hash-Wert erzeugt, der sich auf die gesamte Request-Response-Kette bezieht. Dieser wird aus folgenden Werten generiert:
```

Comment:
"wird jeder Request von dem Kassensystem und auch jeder Response von der fiskaltrust.Middleware gehasht" -->
"wird aus jedem Request des Kassensystems und der zugehörigen Response von der fiskaltrust.Middleware ein Hash-Wert erzeugt"

"wird ein weiterer Hash-Wert erzeugt, der sich auf die gesamte Request-Response-Kette bezieht" --> "wird zeitgleich mit diesem Hash-Wert auch ein weiterer Hash-Wert aus der gesamten, aktuellen Request-Response-Queue erzeugt."

Anstelle der "Kette" könnte man zur einheitlichen Begriffsdefinition gleich "Queue" verwenden. Noch schöner wäre ein neuer Begriff wie zB "ftQueue".

Reply:
Danke fürs Feedback, ich habe es in meinem aktuellen Commit berücksichtigt: https://github.com/christian-rogobete/productdescription-de-doc/commit/2805ee1b9398e0f0b6436d027e64f58da528cd86